src/components/transcript/index.tsx

```diff
@@ -60,6 +67,58 @@ export type sideBarContentUpdateParams<T, K> = {
   name: K;
 };
 
+const getTranscriptContent = (content: TranscriptContent) => {
+  const {
+    speakers,
+    categories,
+    tags,
+    title = "",
+    loc = "",
+    date,
+    ...restContent
+  } = content;
+  const data: SideBarData = {
+    list: {
+      speakers: reconcileArray(speakers),
```

Comment:
I know that this is from previous code, but what's the purpose of the `reconcileArray` function?